plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/sdk/server/JDK.java

```diff
@@ -296,7 +317,8 @@ private static void initialize() {
         minor = subminor = update = 0;
         
         try {
-            String jv = System.getProperty("java.version");
+            String javaVersion = System.getProperty("java.version");
+            vendor = System.getProperty("java.vendor");
```

Comment:
`System.getProperty("java.vendor")` returns `Azul Systems, Inc.` for Zulu11.35+15-CA (build 11.0.5+10-LTS) but `jvm-options` in domain.xml prefixed with `Azul`. Please clarify.

Reply:
@jGauravGupta when checking for the vendor rather than matching what is returned by `System.getProperty("java.vendor")`, it checks if it contains the vendor string prefixed to a JVM option.